cns/restserver/api_test.go

```diff
@@ -588,13 +601,26 @@ func TestGetNetworkContainerVersionStatus(t *testing.T) {
 		podNamespace: "testpodnamespace",
 	}
 
+	// shchen
 	mnma.GetNCVersionF = func(_ context.Context, _ nmagent.NCVersionRequest) (nmagent.NCVersion, error) {
 		return nmagent.NCVersion{
 			NetworkContainerID: params.ncID,
 			Version:            "0", // explicitly 1 less than the version above
 		}, nil
 	}
 
+	//shchen
+	// mnma.GetNCVersionListF = func(_ context.Context) (nmagent.NCVersionList, error) {
+	// 	return nmagent.NCVersionList{
+	// 		Containers: []nmagent.NCVersion{
+	// 			{
+	// 				NetworkContainerID: params.ncID,
+	// 				Version:            "0", // explicitly 1 less than the version above
+	// 			},
+	// 		},
+	// 	}, nil
+	// }
+
```

Comment:
Please remove the commented-out code. We can find it and bring it back with `git` if we need to.